src/modules/cp/const/requests.const.ts

```diff
@@ -53,5 +55,16 @@ export const requestStatusFilters = (): TabProps[] => {
 };
 
 export const requestCreatedAtFormatter = (date: Date): string => {
```

Comment:
is this date the createdAt date?
then consider naming it: createdAtDate

Reply:
This is in line with other constants right now but in general, prefixing these small functions with their context-object is desirable.

Going forward I expect the following;

`src/modules/shared/const/dates.ts`
```
const formatDateToFivePreviousDays = (...) => {...}
```

`src/modules/cp/const/requests.const.ts`
```
const requestCreatedAtFormatter = formatDateToFivePreviousDays;
```

`src/modules/some/other/module/const/foobar.const.ts`
```
const foobarPropertyFormatter = formatDateToFivePreviousDays;
```

Reply:
Personally `requestCreatedAtDateFormatter` seems more descriptive, it lets you know the formatter is intended to be used on dates.